--- a/Cargo.toml	1969-12-31 18:00:00.000000000 -0600
+++ b/Cargo.toml	2021-04-11 21:38:39.267315995 -0500
@@ -140,16 +140,16 @@
 version = "0.5"
 
 [features]
-alsa_backend = ["librespot-playback/alsa-backend"]
+#alsa_backend = ["librespot-playback/alsa-backend"]
-cover = ["ioctl-rs"]
+#cover = ["ioctl-rs"]
 default = ["share_clipboard", "pulseaudio_backend", "mpris", "notify", "cursive/pancurses-backend"]
 mpris = ["dbus", "dbus-tree"]
 notify = ["notify-rust"]
-portaudio_backend = ["librespot-playback/portaudio-backend"]
+#portaudio_backend = ["librespot-playback/portaudio-backend"]
 pulseaudio_backend = ["librespot-playback/pulseaudio-backend"]
-rodio_backend = ["librespot-playback/rodio-backend"]
+#rodio_backend = ["librespot-playback/rodio-backend"]
 share_clipboard = ["clipboard"]
 share_selection = ["clipboard"]
-termion_backend = ["cursive/termion-backend"]
+#termion_backend = ["cursive/termion-backend"]
 [badges.maintenance]
 status = "actively-developed"
