--- a/Cargo.toml	1969-12-31 19:00:01.000000000 -0500
+++ b/Cargo.toml	2022-01-07 00:32:18.207040232 -0500
@@ -151,18 +151,18 @@
 optional = true
 
 [features]
-alsa_backend = ["librespot-playback/alsa-backend"]
+#alsa_backend = ["librespot-playback/alsa-backend"]
 cover = ["ioctl-rs"]
 default = ["share_clipboard", "pulseaudio_backend", "mpris", "notify", "pancurses_backend"]
 mpris = ["dbus", "dbus-tree"]
 notify = ["notify-rust"]
 pancurses_backend = ["cursive/pancurses-backend"]
-portaudio_backend = ["librespot-playback/portaudio-backend"]
+#portaudio_backend = ["librespot-playback/portaudio-backend"]
 pulseaudio_backend = ["librespot-playback/pulseaudio-backend"]
-rodio_backend = ["librespot-playback/rodio-backend"]
+#rodio_backend = ["librespot-playback/rodio-backend"]
 share_clipboard = ["clipboard"]
 share_selection = ["clipboard", "wl-clipboard-rs"]
-termion_backend = ["cursive/termion-backend"]
+#termion_backend = ["cursive/termion-backend"]
 wayland_clipboard = ["wl-clipboard-rs"]
 [badges.maintenance]
 status = "actively-developed"
